loadtest/locustfile.py

```diff
@@ -0,0 +1,72 @@
+from locust import (
+    HttpUser,
+    SequentialTaskSet,
+    TaskSet,
+    task,
+    between
+)
+
+'''
+Para ejecutar los tests de carga:
+    1. Vaciar tablas de la base de datos 
+    2. Para poblarla con datos estáticos, ejecutar el siguiente comando: python manage.py loaddata "fixtures/initial.json"
+'''
+
+HOST = "http://localhost:8000"
+PARTICIPANT_ID = 2
+EVENT_ID = 1
+
+class DefHome(TaskSet):
+
+    @task
+    def index(self):
+        self.client.get("")
+
+class DefLogin(TaskSet):
+
+    @task
+    def index(self):
+        self.client.get("/login")
+
+class DefRegister(TaskSet):
+
+    @task
+    def index(self):
+        self.client.get("/registro")
+
+class DefParticipantProfile(TaskSet):
+
+    @task
+    def index(self):
+        self.client.get("/participante/{0}".format(PARTICIPANT_ID))
+
+class DefEventDetails(TaskSet):
+
+    @task
+    def index(self):
+        self.client.get("/evento/{0}".format(EVENT_ID))
+
+class Home(HttpUser):
+    host = HOST
+    tasks = [DefHome]
+    wait_time = between(3,5)
+
+class Login(HttpUser):
+    host = HOST
+    tasks = [DefLogin]
+    wait_time = between(3,5)
+
+class Register(HttpUser):
+    host = HOST
+    tasks = [DefRegister]
+    wait_time = between(3,5)
+
+class ParticipantProfile(HttpUser):
```

Comment:
This test doesn't work

Reply:
True, I just saw that it is an error that is also in the main branch, I solve it and upload the changes